praw/helpers.py

```diff
@@ -111,6 +115,119 @@ def valid_redditors(redditors, sub):
             if resp['ok']]
 
 
+def all_submissions(reddit_session,
```

Comment:
@voussoir That's a great point. Maybe we can call it `submissions_between`? Would that satisfy your concern @voussoir?

What do you think @eleweek?